copyright/headers/copyright.js.txt

```diff
@@ -0,0 +1,3 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
```

Comment:
Can we make the year dynamic?  It'd be a pain to have to update this every year. :)  When we run the tool it asserts the copyright as of the current year, I think that'd be enough.